l10n_es_aeat_mod303/i18n/es.po

```diff
@@ -1060,36 +1059,3 @@ msgstr "¿SII voluntario?"
 #: model:ir.model.fields,field_description:l10n_es_aeat_mod303.field_l10n_es_aeat_mod303_report__has_operation_volume
 msgid "¿Volumen de operaciones?"
 msgstr "¿Volumen de operaciones?"
-
-#~ msgid "Number of messages which requires an action"
```

Comment:
This msgid was duplicating another one found in line 707.

These are outdated translations, which you can remove with POEdit automatically using this option:

![imagen](https://github.com/OCA/l10n-spain/assets/973709/13c3b081-680d-4cb4-ab99-4a97704badf8)

Take note and add to your muscle memory to do that always before saving a new translation file. Otherwise, if these duplicates slip in, we can hit https://github.com/odoo/odoo/issues/32801 while installing/upgrading the module.